backend/helper_zero/serializers.py

```diff
@@ -1,8 +1,17 @@
 from rest_framework import serializers
 from .models import Organization
+from .models import DonationRequest
+
 
 class OrganizationSerializer(serializers.ModelSerializer):
```

Comment:
Can you add `donation_requests = DonationRequestsSerializer(many=True)` here so it returns donation_requests in the response

Reply:
only for get request tho